cross/node/PLIST.bak

```diff
@@ -0,0 +1,10 @@
+bin:usr/local/bin/node
+bin:usr/local/bin/npm
```

Comment:
As far as there are too many lines in current PLIST, I prefer that version of PLIST file (with wildcard). Please use it instead, after removal of `usr/local` and replacement of `bin` by `rsc` for `npm` and `npx`.